modules/tinymce/src/themes/silver/main/ts/ui/dialog/Collection.ts

```diff
@@ -60,7 +61,9 @@ export const renderCollection = (spec: CollectionSpec, providersBackstage: UiFac
       const ariaLabel = itemText.replace(/\_| \- |\-/g, (match) => {
         return mapItemName[match];
       });
-      return `<div class="tox-collection__item" tabindex="-1" data-collection-item-value="${escapeAttribute(item.value)}" title="${ariaLabel}" aria-label="${ariaLabel}">${iconContent}${textContent}</div>`;
+
+      const readonlyClass = providersBackstage.isReadonly() ? ' tox-collection__item--state-disabled' : '';
```

Comment:
This probably should be using `DomFactory` or something instead, as this is getting quite messy and has a lot of potential for HTML injection.